From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Linus Walleij <linus.walleij@linaro.org>
Date: Mon, 24 Nov 2025 15:35:16 +0100
Subject: pinctrl: cix: Fix obscure dependency

When compile-testing for UM-Linux the build fails because
we don't have IOMEM.

Add an explicit dependency.

Fixes: 920500c5fe66 ("pinctrl: cix: Add pin-controller support for sky1")
Reviewed-by: Peter Chen <peter.chen@cixtech.com>
Signed-off-by: Linus Walleij <linus.walleij@linaro.org>
---
 drivers/pinctrl/cix/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/pinctrl/cix/Kconfig b/drivers/pinctrl/cix/Kconfig
index 111111111111..222222222222 100644
--- a/drivers/pinctrl/cix/Kconfig
+++ b/drivers/pinctrl/cix/Kconfig
@@ -9,6 +9,7 @@ config PINCTRL_SKY1_BASE
 config PINCTRL_SKY1
 	tristate "Cix Sky1 pinctrl driver"
 	depends on ARCH_CIX || COMPILE_TEST
+	depends on HAS_IOMEM
 	select PINCTRL_SKY1_BASE
 	help
 	  Say Y here to enable the sky1 pinctrl driver
-- 
Cixtech

